experimental/alp/lib/Transforms/modulo_scheduling_pass.cpp

```diff
@@ -170,6 +172,7 @@ namespace{
 };
 } // namespace
 
-std::unique_ptr<mlir::FunctionPass> mlir::createModuloSchedulingPass() {
+std::unique_ptr<mlir::OperationPass<FuncOp>>
+mlir::createModuloSchedulingPass() {
```

Comment:
I would similarly hope we use the pipelining pas, could you share IR before and after so it is easier to follow what is going on?

Reply:
Thanks! Added it to my TODO stack.